### ISSUE TITLE
Strings inconsistency in hazards

### ISSUE THREAD

**Issue by gregoriopellegrino** [w3c/publ-a11y#684]
While woking on #618  I found:

- `hazards-flashing-none`: in guidelines "No flashing hazard", in techniques "No flashing hazards"
- `hazards-flashing-unknown`: in guidelines "Flashing hazard not known", in techniques "Flashing hazards not known"
- `hazards-motion-none` (compact): cannot find it in guidelines

@GeorgeKerscher @mattgarrish

**Comment by gregoriopellegrino** [comment]
There is still an inconsistency: in the guidelines "Motion simulation hazards not known," in the techniques "Motion simulation hazards not known," the difference is the plural/singular "hazards/hazards."

All the others seem plural to me.
